### PR TITLE
Add the Slopinator to the review panel

### PR DESCRIPTION
**Pull request by not-stbenjam**
## Motivation

Review conversations leak into shipped code. skillsaw PR #451 merged four
comments that only made sense next to the review thread, and all four had to be
removed afterward:

| Shipped comment | Problem |
|---|---|
| `# Without this a Codex catalog fell through to the single-page renderer` | Describes a bug that never existed on `main` — a reviewer's hypothetical frozen into a permanent claim about history |
| `# this was 99.2% of extract_docs runtime on a 180-plugin repository` | A benchmark from one review round, stale the moment the code changes |
| `# The direct probes were fixed; the recursive scan was not` | Narrates the order the fixes landed in |
| Test sections headed `Review follow-ups, round three` / `round four` | Organizes tests by when they were written, not what they cover |

No existing specialist looks for this. The Technical Writer checks whether docs
are accurate and current; nobody checks whether the writing describes the code
that shipped or the process that produced it.

## What this adds

A seventh specialist, the **Slopinator**. It reads the change as an editor
would, across two halves. A palimpsest — a manuscript whose erased earlier text
still shows through the new writing — is what an AI-authored diff reads like,
and the scope file uses that as its framing. Slop is a defect, so the reviewer
files `BLOCKING` on the same scale as every other specialist.

**Part A — review-history residue in code.** Phantom bugs, round-scoped
benchmark numbers, fix-sequence narration, round-numbered test organization,
change announcements (`# now also handles X`), prose restatements of the next
line, defensive over-commenting, signature-echo docstrings, conversational
filler, changelog narration inside a module. The working heuristic is a
*stranger test*: would this read as a description of the code to someone opening
the file a year from now who has never seen the PR?

**Part B — humanizing technical writing.** Applied to docs, docstrings, commit
messages and PR prose: AI vocabulary tells, copula avoidance ("serves as",
"boasts"), rule-of-three padding, negative parallelism, signposting, sycophancy,
hedging, em dash and boldface abuse, Title Case headings, manufactured
punchlines, generic conclusions, chatbot artifacts. Rewrite suggestions must
never invent a fact, number, name or rationale that is not in the source or the
code.

## Prior art

Part B draws on the [humanizer skill](https://github.com/blader/humanizer) by
blader, which catalogues 33 anti-patterns of AI-generated prose, and on
[Wikipedia:Signs of AI
writing](https://en.wikipedia.org/wiki/Wikipedia:Signs_of_AI_writing) from
WikiProject AI Cleanup, which humanizer is derived from. Both target
encyclopedic and marketing prose. This scope keeps the patterns that survive the
move to a codebase and drops the ones that do not. Humanizer's §30
"Diff-Anchored Writing" is the closest existing statement of the Part A problem;
this reference file expands it into the code-comment case with the concrete #451
examples. The citation is in the reference file's Reference section.

## Calibration — this is not meant to be pedantic

Blocking raises the cost of a false positive, so the scope file is explicit
about restraint:

- `BLOCKING` covers slop, and slop does not merge: an unresolved Slopinator
  `BLOCKING` rules out `APPROVE` on its own and holds the verdict at
  `REQUEST_CHANGES`. It covers **every** Part A residue instance (residue is
  binary, so one review-history comment is enough), any comment or docstring
  that is **factually false** about the shipped code, and Part B tells
  clustered densely enough that a passage reads as generated. Findings must
  quote both the offending text and the proposed replacement, so one edit
  clears them.
- The arbiter may not soften slop — it is not filtered as a style nit and not
  downgraded because the rest of the change is good.
- `SUGGESTION` and `NOTE` remain for a lone prose tell and for one-line polish.
- Clustering governs **Part B only**, where one overused word is genuinely
  nothing. It is never a reason to let residue through.
- A full **"What NOT to flag"** section protects: long "why" comments that earn
  their length, project house style (skillsaw's own files use em dashes and bold
  bullet lead-ins throughout), domain vocabulary and terms of art, a single em
  dash used correctly, version-scoped documents like changelogs and migration
  guides, comments a reviewer explicitly asked for, and existing text a diff
  merely moves.
- Output is capped: prefer clusters over isolated hits, report the strongest
  handful and group the rest as one `NOTE` rather than filing one per instance.
- "Clean writing is a valid outcome" — the file says so, matching the panel's
  existing rule against manufacturing findings.

## Changes

- `references/slopinator.md` — new specialist scope file
- `SKILL.md` — roster row, `description` frontmatter (6 → 7 specialists), the
  "6 specialist reviewers + 1 arbiter" line, parallel/serial dispatch counts and
  roster order, severity calibration (slop added to `BLOCKING`), quality-gate
  checklist row, and the slop carve-out from the panel's advisory stance —
  disposition criteria and arbiter biases now both state that slop does not
  merge
- `references/serial-mode.md` — roster order
- `verdict-template.md` — `{{SLOPINATOR_FINDINGS}}` section
- `DEVELOPMENT.md` — reviewer count and list
- `.agents/` and `.claude/` copies regenerated via `make update`; `make
  verify-apm` confirms they match the `.apm/` source

## Verification

- `make test` — 2965 passed, unchanged from `main` (this PR ships no test
  changes; the panel is a prompt, so its behavior is checked by running it, not
  by asserting on its text)
- `make lint` — clean
- `make update` / `make verify-apm` — all three skill copies identical
- `.venv/bin/skillsaw lint .` — 0 errors, 0 warnings, grade A+

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by not-stbenjam** [comment]
Heads up — this conflicts with #459 on one line, in all three skill copies.

#459 fixes a live bug in the parallel dispatch added by #458: the specialists are launched with `run_in_background: true`, which returns immediately, so the dispatching turn ends with every review still running. Headlessly that ends the CI job, and the panel has posted no verdict since #458 landed ([run 30216174198](https://github.com/stbenjam/skillsaw/actions/runs/30216174198) exited green after 3m22s with nothing posted; a retry reproduced it).

Both PRs edit the same line — this one changes `6` → `7`, #459 changes `true` → `false`. Resolution is **`all 7` + `run_in_background: false`**.

Suggest landing #459 first, since until it does the Palimpsest Reviewer would be dispatched and abandoned along with the other six.

**Comment by stbenjam** [comment]
Fix conflicts and Palimpsest is a weird name. Call it slopinator.  I think you made this from an outdated branch ...

File: .apm/skills/skillsaw-review-panel/SKILL.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ the maintainer and PR author review and decide ship.
 | QA Engineer | Test coverage gaps, untested error paths, edge cases, concrete test suggestions | [`references/qa-engineer.md`](references/qa-engineer.md) |
 | Technical Writer | Verify documentation accuracy, completeness, consistency with code changes, CLAUDE.md drift | [`references/technical-writer.md`](references/technical-writer.md) |
 | Ecosystem Reviewer | Review target-tool adoption in the current LLM landscape; core-vs-plugin scope boundary | [`references/ecosystem.md`](references/ecosystem.md) |
+| Palimpsest Reviewer | Catch review-history residue in code comments and AI-authored prose tells in docs, docstrings, and commit messages | [`references/palimpsest.md`](references/palimpsest.md) |
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply .apm/skills/skillsaw-review-panel/SKILL.md]: Added in 4193ec7 — `tests/test_review_panel_skill.py`, 31 tests. The panel is a prompt rather than code, so its failure modes are textual and invisible to the rest of the suite; that is precisely how the headless silent-exit in #458 shipped green.

Coverage: every specialist is rostered with a scope file that exists and a quality-gate line; the declared count matches the roster in all three places it appears; the verdict template has a section per specialist; Palimpsest reports a candidate rather than deciding its own severity; the arbiter has the settle step; and all three skill copies are byte-identical, which caught a real desync while I was working.

File: .apm/skills/skillsaw-review-panel/SKILL.md [hunk 2]
```diff
@@ -117,6 +118,11 @@ If no issues found, say so and list what was checked.
   correctness issue. Keep this the default for real feedback.
 - `NOTE`: One-line polish, style nits, minor improvements.
 
+The Palimpsest Reviewer is advisory: its findings are `SUGGESTION` or `NOTE`
+by default and never set the disposition on their own. It may only reach
+`BLOCKING` when a comment or docstring is factually false about the shipped
+code, and another specialist corroborates the underlying issue.
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply .apm/skills/skillsaw-review-panel/SKILL.md]: Good catch — this was a genuine logic hole, not a wording nit. The condition was unsatisfiable by construction: specialists are isolated, so Palimpsest could never observe corroboration, would always take the downgrade branch, and synthesis had no step to promote it back. A corroborated correctness defect could ship as a SUGGESTION under an APPROVE.

Fixed in 4193ec7 exactly as you suggested. It now reports `BLOCKING CANDIDATE`, states what corroboration would confirm it, and is explicitly told not to downgrade itself. A new synthesis step 5 has the arbiter promote it when Architecture or the Technical Writer reached the same claim, otherwise record it as SUGGESTION — and says leaving it unsettled is not an option, since an unpromoted candidate silently becomes advisory. Both halves are pinned by tests.

File: .agents/skills/skillsaw-review-panel/SKILL.md [hunk 3]
```diff
@@ -49,6 +49,7 @@ the maintainer and PR author review and decide ship.
 | QA Engineer | Test coverage gaps, untested error paths, edge cases, concrete test suggestions | [`references/qa-engineer.md`](references/qa-engineer.md) |
 | Technical Writer | Verify documentation accuracy, completeness, consistency with code changes, CLAUDE.md drift | [`references/technical-writer.md`](references/technical-writer.md) |
 | Ecosystem Reviewer | Review target-tool adoption in the current LLM landscape; core-vs-plugin scope boundary | [`references/ecosystem.md`](references/ecosystem.md) |
+| Palimpsest Reviewer | Catch review-history residue in code comments and AI-authored prose tells in docs, docstrings, and commit messages | [`references/palimpsest.md`](references/palimpsest.md) |
 | Panel Arbiter | Handle strategic synthesis, disagreement resolution, final disposition | *(inline, below)* |
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply .agents/skills/skillsaw-review-panel/SKILL.md:52]: Checked, and there is nothing to update: `README.md` does not mention the review panel or any other skill — `grep -c "skillsaw-review-panel\|Review Panel" README.md` returns 0. Its sections are See it work / Try it / What it catches / Built for real workflows / Measure the result / Contributing, all about the linter itself. The skills are documented in `DEVELOPMENT.md`, which this PR does update with the new specialist.

Adding a README section for one specialist of one skill would introduce the inconsistency rather than fix it. Happy to document the skills in the README as its own change if that is wanted.

**Comment by stbenjam** [comment]
Rename the reviewer to slopinator.

File: .apm/skills/skillsaw-review-panel/references/palimpsest.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+it describes the code that shipped or the process that produced it.
+
+**This reviewer is advisory and must never be the sole reason a PR is blocked.**
+Findings are `SUGGESTION` or `NOTE` by default. See Severity calibration below
+for the one narrow case that escalates.
+
+## Part A — Review-history residue in code
+
+The motivating case, from a real review on skillsaw PR #451. All four shipped
+and had to be removed afterward:
```
Comment (stbenjam) [inline .apm/skills/skillsaw-review-panel/references/palimpsest.md]: Don't reference the PR

File: .apm/skills/skillsaw-review-panel/SKILL.md [hunk 5]
```diff
@@ -117,6 +118,11 @@ If no issues found, say so and list what was checked.
...
+
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply .apm/skills/skillsaw-review-panel/SKILL.md]: Fixed in 2317ac9. All three places now state one rule:

> Corroboration means another specialist independently reports the same underlying defect — the same wrong behavior or false claim about the code, not merely a finding in the same file.

Agreed that "same file" was too weak, so that phrasing is gone. I also dropped the named-specialist gate: restricting corroboration to Architecture or the Technical Writer was arbitrary — a QA or Security finding about the same wrong behavior corroborates just as well — and it made the rule narrower in one place than in the other two. It is now "any other specialist" everywhere.

- `SKILL.md` severity list — the canonical statement.
- `references/palimpsest.md` severity calibration — same wording, from the specialist's side.
- `SKILL.md` Step 5 settle step — "Promote to `BLOCKING` when any other specialist independently reported the same underlying defect — the same wrong behavior or false claim, not merely a finding in the same file."

`TestCorroborationContractIsConsistent` asserts each of the three carries the definition, and `test_corroboration_is_not_narrowed_to_named_specialists_or_to_a_file` fails if `reaches the same file`, `reached the same claim`, or either named-specialist phrasing comes back.

File: .apm/skills/skillsaw-review-panel/SKILL.md [hunk 6]
```diff
@@ -163,27 +165,14 @@ If the Agent tool is not available (e.g. running in Codex or another
 client that lacks sub-agent support), fall back to serial mode
 automatically.
 
-Synchronous dispatch means all six have returned by the time the
-tool call completes. Proceed to the Completeness Gate (Step 4) only
-with six sets of findings in hand.
+Synchronous dispatch means all have returned by the time the call
+completes. Proceed to Step 4 only with every specialist's findings.
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply .apm/skills/skillsaw-review-panel/SKILL.md]: Correct, and fixed in 2317ac9. The intent was "don't read results that are still arriving", not "all seven must succeed", and the old wording said the latter.

The sentence is gone. The dispatch paragraph now ends:

> Do not start Step 4 while results are still arriving; a specialist that errors is handled there, not waited on.

That points at the retry-then-record-and-proceed path instead of contradicting it, and Step 4's "post the verdict anyway, naming which failed" is unchanged. `TestDispatchGateDoesNotStallOnFailure` pins both halves: the gate is about arrival, and a failed specialist still yields a verdict.

Folding this into the existing `run_in_background` paragraph also removed a duplicate statement of the same fact, which helped with the context-budget thread below.

File: DEVELOPMENT.md [hunk 7]
```diff
@@ -138,7 +138,7 @@ The project includes a review panel skill for thorough PR review:
 /skillsaw-review-panel                # review the current local branch
 ```
 
-This runs 6 specialist reviewers (architecture, Python, security, QA, docs, ecosystem) as parallel sub-agents plus an arbiter, and posts a structured verdict as a PR comment. Use `--serial` for cheaper inline execution.
+This runs 7 specialist reviewers (architecture, Python, security, QA, docs, ecosystem, palimpsest) as parallel sub-agents plus an arbiter, and posts a structured verdict as a PR comment. Use `--serial` for cheaper inline execution.
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply DEVELOPMENT.md]: Right — the line predates this PR but the sentence I edited made it worse by leaving the claim unqualified. Fixed in 2317ac9:

> This runs 7 specialist reviewers (architecture, Python, security, QA, docs, ecosystem, palimpsest) as parallel sub-agents plus an arbiter. Reviewing a PR posts a structured verdict as a PR comment; reviewing the current local branch prints the verdict instead.

That matches Step 6, which posts one comment when a PR number is known and outputs to the user otherwise.

File: .apm/skills/skillsaw-review-panel/SKILL.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply .apm/skills/skillsaw-review-panel/SKILL.md]: Good catch, and the diagnosis is exactly right: the findings format is the entire contract the isolated sub-agent gets, so a label absent from it cannot be emitted, and the arbiter's settle step would have had nothing to settle. Fixed in 2317ac9.

The shared severity line is now:

> - **Severity**: `BLOCKING` | `BLOCKING CANDIDATE` | `SUGGESTION` | `NOTE`

and the calibration list gained a bullet scoping it:

> - `BLOCKING CANDIDATE`: **Palimpsest Reviewer only** — no other specialist uses it. That reviewer is advisory: `SUGGESTION` or `NOTE` by default, never setting the disposition itself. It reports this label for a comment that is factually false about the shipped code, rather than deciding a severity it cannot judge alone. Only the arbiter promotes it (Step 5).

`TestFindingsFormatCarriesEverySeverity` asserts the label is in the shared list, that it is marked Palimpsest-only, and that promotion is the arbiter's.

File: tests/test_review_panel_skill.py [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+class TestDispatchIsAwaited:
+    """A background dispatch ends the turn, which ends a headless CI job.
+
+    The workflow then exits green having posted no verdict — indistinguishable
+    from a clean review unless someone reads the execution artifact.
+    """
+
+    def test_parallel_dispatch_is_synchronous(self):
+        body = skill_md(SOURCE)
+        assert "`run_in_background: false`" in body
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply tests/test_review_panel_skill.py]: Fair, and I won't re-argue it. Added a real fixture in 2317ac9: `tests/fixtures/palimpsest-review-residue/`, a small repo whose content is what the reviewer is meant to catch, beside what it must leave alone.

| File | Role |
|---|---|
| `src/catalog.py` | Every Part A pattern: the phantom-bug comment, the 99.2% benchmark, the fix-sequence narration, a change announcement, prose restatements, defensive over-commenting, a signature-echo docstring, conversational filler, changelog narration |
| `tests/catalog_test.py` | `Review follow-ups, round three` / `round four` sections |
| `docs/overview.md` | The Part B tells, clustered as they actually occur |
| `src/renderer.py` | Clean: genuine "why" comments, including a long one that earns its length |
| `docs/reference.md` | Clean: domain vocabulary, house-style bold bullets, one correctly used em dash |
| `CHANGELOG.md` | Clean: a version-scoped document, where narrating change is correct |

The tests pin the fixture to the scope file in **both** directions. `bullet_terms()` parses the bold lead-ins out of each list in `references/palimpsest.md`, and `test_the_fixture_covers_every_pattern_the_scope_file_names` asserts set equality against the marker table. Adding a pattern to the scope file without an example here fails; adding an example for a pattern the scope file no longer names also fails. `test_the_clean_files_carry_no_residue` asserts none of the 22 residue markers leaked into the three clean files.

**What stays un-automated, and why.** Nothing here asserts the reviewer actually catches any of it. The panel is a prompt executed by an LLM: recall and precision need a model in the loop, are nondeterministic, and would make the unit suite slow and flaky. So the fixture is coverage a human or a live `/skillsaw-review-panel` run can be pointed at — the fixture README says so and states the calibration check plainly: a reviewer that flags anything in `renderer.py`, `reference.md`, or `CHANGELOG.md` is miscalibrated.

Two entries in "What NOT to flag" are properties of the review *situation*, not of any file — "comments a reviewer explicitly requested" needs a PR thread, and "existing text the diff merely moves" needs a diff. No static fixture can carry those. Rather than quietly narrowing the coverage assertion, they are listed in `UNREPRESENTABLE_CLEAN_CASES` and the set-equality test includes them, so a new "what not to flag" rule has to be either exercised or explicitly declared un-representable.

The fixture test file uses the `*_test.py` name so this repo's `python_files = ["test_*.py"]` does not collect fixture material as real tests.

File: .apm/skills/skillsaw-review-panel/SKILL.md [hunk 9]
```diff
@@ -210,8 +198,13 @@ After all specialists complete, review and synthesize directly:
    findings, and issues already addressed in the branch.
 4. **Resolve conflicts** — corroboration strengthens; when specialists
    disagree, prefer the more conservative position.
-5. Set a disposition (see below).
-6. Include required actions (blocking) vs optional follow-ups.
+5. **Settle blocking candidates** — Palimpsest reports a factually false
+   comment as a `BLOCKING CANDIDATE`, since it cannot see other specialists.
+   Promote to `BLOCKING` when Architecture or the Technical Writer reached
+   the same claim; otherwise record as `SUGGESTION`. Leaving it unsettled is
+   not an option.
```
[the remark this answers is not in the document]
Comment (not-stbenjam) [reply .apm/skills/skillsaw-review-panel/SKILL.md]: Confirmed and fixed in 2317ac9. Two separate budget problems, both cleared.

**`content-section-length`** — putting the disposition criteria under their own `#### Disposition criteria` heading splits the arbiter step, which was ~546 tokens as one section. Gone.

**`context-budget`** — worth flagging, because the fixes for the other threads made it worse before it got better. Adding the corroboration definition and the `BLOCKING CANDIDATE` bullet pushed `SKILL.md` to 3,083 tokens, over the 3,000 warn limit, i.e. a genuine WARNING and not only an info. I paid for it out of duplication rather than content:

- The `run_in_background` explanation and the "synchronous dispatch" paragraph stated the same fact twice; merged (this is also the thread-2 fix).
- "Each specialist runs independently and cannot see the others' output" appeared three times once the candidate bullet restated it; kept once.
- The full scope-file path was spelled out in both "Prompt path resolution" and the sub-agent bullet list; the bullet now refers to the pattern above.
- Condensed the two-mode intro, which duplicated the Arguments table.

`SKILL.md` is 2,967 tokens — under the limit, and 21 tokens *below* what it was before this thread's fixes. `TestContextBudget` keeps it pinned.

Full-repo comparison against `origin/main`, both at `-v`:

| | errors | warnings | info |
|---|---|---|---|
| `origin/main` | 0 | 0 | 6 |
| this branch | 0 | 0 | 6 |

Same six, same rules, same files — 4 × `content-actionability-score`, 2 × `content-inconsistent-terminology`, all pre-existing. The regenerated `.skillsaw-card.svg` dropped its `content-section-length (1)` row and density went 0.21 → 0.19 per 10k tokens. The new fixture is not linted: `tests/fixtures/*` is excluded in `.skillsaw.yaml`, which is the existing convention for fixtures with intentional defects.

File: .apm/skills/skillsaw-review-panel/references/palimpsest.md (hunk 4 above)
Comment (not-stbenjam) [reply .apm/skills/skillsaw-review-panel/references/palimpsest.md]: Fixed in cec6418 — and fair, the scope file was doing the thing it tells the reviewer to flag. Citing the review the examples came from is diff-anchored writing; a reader a year from now gets nothing from the number.

The intro is now:

> Comments of this shape are the motivating case. Each says something true about a review conversation and nothing useful about the code:

The four example comments stay, since they illustrate the pattern without needing the provenance. `#451` no longer appears anywhere in the skill, the fixture, or the tests — only in the PR description, which is version-scoped by nature.

**Comment by not-stbenjam** [comment]
Renamed to **Slopinator** in 5006f28 — scope file (`references/slopinator.md`), fixture directory (`tests/fixtures/slopinator-review-residue/`), the `{{SLOPINATOR_FINDINGS}}` verdict placeholder, and every mention in the roster, serial-mode reference, tests, and DEVELOPMENT.md. All three skill copies stay byte-identical.

The scope file used to open by explaining the palimpsest metaphor, which the new name does not carry, so that paragraph now states the residue problem directly instead.

On the conflict: #459 landed at 19:19 and is already this branch's merge base, so there is nothing left to resolve. The merged text is the resolution that was suggested — `all 7 specialist sub-agents` with `run_in_background: false`.

Full suite green (3043 passed), `black --check` clean, self-lint A+; report card regenerated.

File: tests/test_review_panel_skill.py
Comment (stbenjam) [inline tests/test_review_panel_skill.py:1]: Remove the tests here, @codex suggestions for testing a skill like this are total nonsense.

Skills are tested with LLM-enabled evals, not unit tests! (Don't add evals either)

Comment (not-stbenjam) [reply tests/test_review_panel_skill.py:1]: Removed in bf41439 — `tests/test_review_panel_skill.py` is gone, no evals added.

I also removed `tests/fixtures/slopinator-review-residue/`, the corpus that test file was the only consumer of. Happy to restore it on its own if you want a repo to point a live panel run at for calibration, but it was scaffolding for the tests you just cut.

Suite is back to 2965 passed, the exact `main` baseline, so nothing else depended on either.

**Comment by stbenjam** [comment]
You are too careful, slop is blocking.  Palmpisest-style comments need to never merge.

**Comment by not-stbenjam** [comment]
Slop blocks now, in aef37ac.

Dropped the whole advisory apparatus: the `BLOCKING CANDIDATE` label, the corroboration requirement, and the arbiter's Step 5 that existed only to promote candidates. The Slopinator files `BLOCKING` on the same scale as every other specialist, and slop is listed under `BLOCKING` in the shared severity calibration.

Since a false positive now costs more, the restraint rules got firmer rather than looser — clustering is stated as a requirement, `SUGGESTION`/`NOTE` stay available for a lone tell or one-line polish, and blocking findings must quote both the offending text and the replacement so one edit clears them. "What NOT to flag" is unchanged.

Also brought the palimpsest metaphor back into the scope file as the framing for what the reviewer looks for, just not as the name.

One thing worth naming: `SKILL.md` line 27 has said "Keep the panel **advisory**. It does not gate merge" since before this PR. So a Slopinator `BLOCKING` drives `REQUEST_CHANGES` on the verdict, it does not hard-block the merge button. Say the word if you want that panel-wide line changed too — it is outside this PR's scope so I left it.

2965 passed, self-lint A+, all three skill copies identical.

**Comment by not-stbenjam** [comment]
Fair — it was blocking in name only. Fixed in dfd40c6.

The gap was the panel-wide line that said the panel "does not gate merge", which meant a Slopinator `BLOCKING` could still land under an `APPROVE`. Three changes close it:

- **Slop is carved out of the advisory stance.** An unresolved Slopinator `BLOCKING` rules out `APPROVE` on its own and holds the verdict at `REQUEST_CHANGES`. Stated in the panel intro and in the disposition criteria.
- **The arbiter may not soften it.** New arbiter bias: a slop finding is not filtered as a style nit in the noise-filter step, and not downgraded because the rest of the change is good.
- **Every Part A instance blocks.** Residue is binary — a comment either describes the shipped code or the conversation that produced it — so the clustering rule now explicitly governs Part B prose only. One review-history comment is enough. Palimpsest-style comments do not merge.

Worth flagging so it is not a surprise: the panel posts its verdict with `gh pr comment`, so `REQUEST_CHANGES` is a verdict in a comment, not a GitHub review state. Nothing in this skill can physically grey out the merge button. If you want a hard gate, the panel would need to post an actual review (`gh pr review --request-changes`) with branch protection requiring approval — happy to do that here or in a follow-up, just say which.

2965 passed, self-lint back to 6 info violations, the exact `main` baseline (my first pass had pushed "What NOT to flag" over the section-length limit; split the reporting-volume guidance into its own section). All three skill copies identical.
